extentions/validations.rb

```diff
@@ -0,0 +1,7 @@
+module Validations
+    private
+
+    def check_instance(param, class_instance, main_class)
```

Comment:
It's better to call it "validate_instance" since you are validating it with this method, not just checking for smth